src/server/src/districts/services/topology.service.ts

```diff
@@ -70,16 +70,18 @@ export class TopologyService {
           }),
           {}
         );
+        // Filter out archived regions
+        const filteredRegions = _.filter(regionConfigs, ["archived", false]);
```

Comment:
Sometimes you need explain what you're doing in a comment because it isn't immediately obvious what the code does, but usually when I see myself writing a "this is what the code does" comment I try and see if renaming some variables / rewriting the code to be more self-explanatory would remove the need for it.

```suggestion
        const unarchivedRegions = _.filter(regionConfigs, region => !region.archived]);
```